client/tailscale/apitype/apitype.go

```diff
@@ -10,9 +10,16 @@ import "tailscale.com/tailcfg"
 const LocalAPIHost = "local-tailscaled.sock"
 
 // WhoIsResponse is the JSON type returned by tailscaled debug server's /whois?ip=$IP handler.
-// In successful whois responses, Node and UserProfile are never nil.
 type WhoIsResponse struct {
-	Node        *tailcfg.Node
+	// Node is the node that the IP address belongs to.
+	//
+	// It is always non-nil in successful responses.
+	Node *tailcfg.Node
+
+	// UserProfile is the profile of the Node.User if the node is not tagged.
```

Comment:
Just for my understanding: If the node _is_ tagged, it sounds like this field is still set (per the "always non-nil" below), but then what _is_ the value of this field in that case?

Reply:
It basically says "tagged-devices"